src/components/Correction/AgreementType.vue

```diff
@@ -22,58 +24,56 @@
             </v-chip>
           </div>
         </div>
-        <div>
-          <v-flex md2 class="align-right">
-            <v-btn v-if="!hasAgreementTypeChange"
-              text
-              color="primary"
-              id="btn-correct-agreement-type"
-              @click="showAgreementTypeForm = true">
-              <v-icon small>mdi-pencil</v-icon>
-              <span>Correct</span>
-            </v-btn>
-            <v-btn v-if="hasAgreementTypeChange"
-              text
-              color="primary"
-              id="btn-undo-agreement-type"
-              @click="resetAgreementType">
-              <v-icon small>mdi-undo</v-icon>
-              <span>Undo</span>
-            </v-btn>
-          </v-flex>
-        </div>
-      </div>
-      <div v-else>
-        <v-card flat>
-          <v-radio-group v-model="agreementType" class="agreement-option-list">
-            <v-radio
-              v-for="(item, index) in AgreementTypeResource"
-              :key="index"
-              :value="item.code"
-              :id="`agreement-type-${item.code}`">
-              <template slot="label">
-                <div v-html="item.description" class="agreement-option" />
-              </template>
-            </v-radio>
-          </v-radio-group>
-          <div class="action-btns">
-            <v-btn id="done-btn" large color="primary" @click="setAgreementType">
-              <span>Done</span>
-            </v-btn>
-            <v-btn id="cancel-btn" large outlined color="primary" @click="resetAgreementType">
-              <span>Cancel</span>
-            </v-btn>
-          </div>
-        </v-card>
+        <v-flex md2 class="align-right">
+          <v-btn v-if="!hasAgreementTypeChange"
+            text
+            color="primary"
+            id="btn-correct-agreement-type"
+            @click="showAgreementTypeForm = true">
+            <v-icon small>mdi-pencil</v-icon>
+            <span>Correct</span>
+          </v-btn>
+          <v-btn v-if="hasAgreementTypeChange"
+            text
+            color="primary"
+            id="btn-undo-agreement-type"
+            @click="resetAgreementType">
+            <v-icon small>mdi-undo</v-icon>
+            <span>Undo</span>
+          </v-btn>
+        </v-flex>
       </div>
+
+      <!-- Edit -->
+      <v-card flat v-else :class="{ 'invalid-section': !agreementType }">
```

Comment:
The changes above and here are to display the component correctly where is there no Agreement Type (including error styling and validation). Example:

![image](https://user-images.githubusercontent.com/10002889/179126873-7f56ed01-91c6-4956-a8a2-00783e64b555.png)